src/app/components/toggle-switch/index.html

```diff
@@ -0,0 +1,200 @@
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+    </sky-demo-page-content>
+
+        <sky-demo-page-content sectionHeading="Anatomy">
+            <sky-docs-anatomy imagePath="~/assets/img/guidelines/toggle/toggle-anatomy.png">
+                <sky-docs-anatomy-item anatomyNumber="1">Toggle body</sky-docs-anatomy-item>
+                <sky-docs-anatomy-item isOptional="true" anatomyNumber="2">Text label</sky-docs-anatomy-item>
+            </sky-docs-anatomy>
+        </sky-demo-page-content>
+
+        <sky-demo-page-content sectionHeading="Options">
+            <h3>Text label</h3>
+            <p>When toggles are in grid columns, the text label is not required if you use a column heading. Use text labels for all other states.
+            </p>
+            <p>If the scenario allows for toggles to become disabled, always include a text label on each toggle.</p>
+        </sky-demo-page-content>
+
+        <sky-demo-page-content sectionHeading="Behavior & states">
+            <sky-demo-page-component-states>
+              <sky-demo-page-component-state stateName="Unselected" imagePath="~/assets/img/guidelines/toggle/toggle-state-unselected.png"></sky-demo-page-component-state>
+              <sky-demo-page-component-state stateName="Selected" imagePath="~/assets/img/guidelines/toggle/toggle-state-selected.png"></sky-demo-page-component-state>
+              <sky-demo-page-component-state stateName="Hover" imagePath="~/assets/img/guidelines/toggle/toggle-state-hover.png"></sky-demo-page-component-state>
+              <sky-demo-page-component-state stateName="Disabled unselected" imagePath="~/assets/img/guidelines/toggle/toggle-state-disabled-unselected.png"></sky-demo-page-component-state>
+              <sky-demo-page-component-state stateName="Disabled selected" imagePath="~/assets/img/guidelines/toggle/toggle-state-disabled-selected.png"></sky-demo-page-component-state>
+            </sky-demo-page-component-states>
+        </sky-demo-page-content>
+
+        <sky-demo-page-content sectionHeading="Related content">
+          <h3>Components</h3>
+      <ul>
+        <li><a routerLink="/components/checkbox">Checkbox</a></li>
+        <li><a routerLink="/components/radio">Radio button</a></li>
+      </ul>
+
+        </sky-demo-page-content>
+    </sky-tab>
+
+    <sky-tab tabHeading="Development">
+
+      <sky-demo-page-module-info
+        skyuxModule="SkyToggleSwitchModule"
+        packageName="@skyux/forms"
+        githubRepo="skyux-forms"
+      >
+      </sky-demo-page-module-info>
+
+      <sky-demo-page-properties
+        sectionHeading="Toggle switch properties"
+      >
+        <sky-demo-page-property
+          propertyName="ariaLabel"
+          [isOptional]="true"
+        >
+          Sets the <sky-code>aria-label</sky-code> attribute of the toggle switch. Only use this value if the `sky-toggle-switch-label` component is not used. This property accepts <sky-code>string</sky-code> values.
```

Comment:
Should sky-toggle-switch-label be between the <sky-code> tag, or are the tick marks intentional?

Reply:
Oops! Good catch.